github.go

```diff
@@ -126,7 +127,9 @@ func FetchTags(githubRepoUrl string, githubToken string, instance GitHubInstance
 	}
 
 	for _, tag := range tags {
-		tagsString = append(tagsString, tag.Name)
+		if _, err := version.NewVersion(tag.Name); err == nil {
```

Comment:
NIT: Add a comment here explaining we want to skip tags that aren't semantically versioned so they don't cause errors. Could even cross-link to the issue you filed.